llarp/quic/tunnel.cpp

```diff
@@ -591,9 +606,34 @@ namespace llarp::quic
     auto conn = tunnel.client->get_connection();
 
     conn->on_stream_available = [this, id = row.first](Connection&) {
-      LogDebug("QUIC connection :", id, " established; streams now available");
+      log::debug(logcat, "QUIC connection :{} established; streams now available", id);
       if (auto it = client_tunnels_.find(id); it != client_tunnels_.end())
+      {
         flush_pending_incoming(it->second);
```

Comment:
maybe change to `this->flush_pending_incoming` to match the style of `this->close` below?

Reply:
(Possibly that's why `this->close` was used instead of just `close`?)

Reply:
`close` is ambiguous with `::close` in this context